doc news: add 14.0.8 entry (#1960)
groonga/groonga: doc/source/news/14.md

# News - 14 series

(release-14-0-8)=
## Release 14.0.8 - 2024-09-25

### Improvements

* [Amazon Linux 2023] Added support for Amazon Linux 2023. [GH-1845][Reported by takoyaki-nyokki and Watson]

  However, we don't support Amazon Linux 2023 for aarch64 currently.

>   Because the maintenance costly because of the time of build for package of aarch64 is too time consuming.

Reviewer: ```suggestion
  Because the maintenance is costly because of the time of building a package for aarch64 is too time consuming.
```

>   We don't use TokenMecab on Groonga for Amazon Linux 2023.

Reviewer: ```suggestion
  We don't use TokenMecab in Groonga for Amazon Linux 2023.
```

  Because Amazon Linux 2023 doesn't provide MeCab package.

### Fixes

> * Fixed a crash bug when we specify `pgroonga.log_type` on PGroonga for Windows.

Reviewer: ```suggestion
* Fixed a crash bug when we specified `pgroonga.log_type` on PGroonga for Windows.
```
